packages/vite-plugin-postgres/src/index.ts

```diff
@@ -6,21 +6,27 @@ import { loadEnv, type Plugin as VitePlugin } from "vite";
 interface PostgresPluginOptions {
 	env: string;
 	envKey: string;
-	referrer: string;
+	onCreate?: {
+		type: "sql-script";
+		path: string;
+	};
 }
 
 const DEFAULTS: PostgresPluginOptions = {
 	env: ".env",
 	envKey: "DATABASE_URL",
-	referrer: "unknown",
+	onCreate: {
+		type: "sql-script",
+		path: "",
+	},
 };
 
 let claimProcessStarted = false;
 
 export default function postgresPlugin(
 	options?: Partial<PostgresPluginOptions>,
 ): VitePlugin {
-	const { env: envPath, envKey, referrer } = { ...DEFAULTS, ...options };
+	const { env: envPath, envKey } = { ...DEFAULTS, ...options };
```

Comment:
No sure what's the reason to remove this parameter. We'll want to be able to distinguish between the different projects/starters that'll use this Vite plugin (like Tanstack Start...). I'd keep it here.

Reply:
that's a user-facing config. So it's less likely that we can we rely on.

E.g.: `@neondatabase:vite-plugin:my-app` instead of the integration layer.
I'll add it back - but I think this can fragment vite-plugin metrics and make them harder to read.

Reply:
Yep but the same is true already with the `instantNeon` function. _Some_ people can always pass 💩 there. But if/when we do another starter/framework than Tanstack, we'll be happy to be able to pass a parameter. And we can always query our analytics with "starts with" queries, not "exact equals".